### PR TITLE
Authorites stats integrations

### PR DESCRIPTION
**Pull request by AhmedKorim**
## Summary of changes
- 

### Reference issue to close (if applicable)
- Closes 

-----
### Code Checklist 

- [ ] Tested
- [ ] Documented

### CODE REVIEW

File: packages/page-statistics/src/containers/AuthorityDetail/AuthorityDetail.tsx [hunk 1]
```diff
@@ -99,39 +72,59 @@ const columns: ColumnDef<KeyGenKeyListItem, any>[] = [
...
-        <Button className='uppercase' varirant='link' as='span' size='sm'>
-          Details
-        </Button>
-      </Link>
-    ),
+    cell: (props) => {
+      const id = props.row.original.publicKey;
+      return (
+        <Link to={`/keys/drawer/${id}`}>
+          <Button className='uppercase' varirant='link' as='span' size='sm'>
```
Comment (nepoche) [inline packages/page-statistics/src/containers/AuthorityDetail/AuthorityDetail.tsx:94]:
```suggestion
          <Button className='uppercase' variant='link' as='span' size='sm'>
```

File: packages/page-statistics/src/provider/hooks/useAuthorities.ts [hunk 2]
```diff
@@ -46,7 +48,7 @@ export type UpcomingThreshold = {
   signature: string;
   proposer: string;
   // TODO use the type `DiscreteList`
```
Comment (nepoche) [inline packages/page-statistics/src/provider/hooks/useAuthorities.ts:50]:
```suggestion
```

**Review by nepoche — APPROVED** [review]
LGTM

**Review by AtelyPham — APPROVED** [review]
LGTM 👍 @AhmedKorim pls fix the ci before merging.

File: packages/page-statistics/src/App.tsx [hunk 3]
```diff
@@ -4,13 +4,16 @@ import { RouterProvider } from '@webb-dapp/react-environment/RouterProvider';
...
+
 import { routes } from './routes';
+
 const isDev = process.env.NODE_ENV === 'development';
 const apolloClient = new ApolloClient({
   cache: new InMemoryCache(),
   uri: isDev ? 'http://localhost:4000' : 'https://api.subquery.network/sq/AtelyPham/webb-subql',
 });
-
+console.log(encodeAddress(`0x69a1e29c03787e72adbd6fc93195376051d974bb545b2a5f42d834e0a0adfb83`));
```
Comment (dutterbutter) [inline packages/page-statistics/src/App.tsx:16]: Can we remove this please @AhmedKorim

**Pull request merged**